core/build.gradle.kts

```diff
@@ -9,6 +9,7 @@ plugins {
   id("antlr")
   id("com.google.protobuf") version "0.8.17"
   id("com.diffplug.spotless") version "6.11.0"
+  id("com.github.johnrengelman.shadow") version "7.1.2"
```

Comment:
Just poking around the docs for this, is there any reason not to use the latest version which is 8.1.1?

Reply:
That would require Gradle v8: https://github.com/johnrengelman/shadow#latest-test-compatibility

Reply:
Ah, that seems like work for a different day then.